src/qt/optionsdialog.cpp

```diff
@@ -406,24 +406,21 @@ void OptionsDialog::updateProxyValidationState()
 
 void OptionsDialog::updateDefaultProxyNets()
 {
+    CNetAddr ui_proxy_netaddr;
+    LookupHost(ui->proxyIp->text().toStdString(), ui_proxy_netaddr, /*fAllowLookup=*/false);
+    const CService ui_proxy{ui_proxy_netaddr, ui->proxyPort->text().toUShort()};
```

Comment:
7f420c508968 TIL about `toUShort()` (https://doc.qt.io/qt-5/qlocale.html#toUShort) used for the first time in this codebase. It seems compatible with our minimum required version of qt 5.11.3.

Reply:
I wasn't aware of `toUShort()` either. IIRC the auto-completion showed it to me ;-)